faraday/uart/layer_2_protocol.py

```diff
@@ -15,14 +15,20 @@
 import time
 import struct
 import logging
+import sys
 
 # Get configured logger
 logger = logging.getLogger('UARTStack')
 
 
 class layer_2_object(object):
```

Comment:
Can we just remove this class entirely? It looks like it doesn't serve any purpose than to hold a layer_2_protocol object.